.github/workflows/on_push.yaml

```diff
@@ -14,3 +14,8 @@ jobs:
     permissions:
       pull-requests: read
     secrets: inherit
+    with:
+      rockcraft-channel: latest/edge
```

Comment:
Regarding rockcraft-channel, ~~if we don't use a specific feature from last `1.1.0`, then I think we could omit this input and go with `stable` here~~
I saw that they have released the fixes in `stable` (see [discussion in matrix](https://matrix.to/#/!NPPCseDHKRvSBMUEXN:ubuntu.com/$sX6hnpqkzKsWurwzUPcXHgLqA8gdOR1mCUi61jTibEc?via=ubuntu.com&via=matrix.org) or [snap](https://snapcraft.io/rockcraft))

Reply:
updated to stable.  I'd pin to a version, but they don't have versioned channels :(